flocker/control/test/test_httpapi.py

```diff
@@ -2968,6 +2968,44 @@ def test_one_container(self):
             b"GET", b"/state/containers", None, OK, response
         )
 
+    def test_one_container_maximum_size(self):
+        """
+        When the cluster state includes one container that has a dataset with
+        a maximum size, the endpoint returns a single-element list
+        containing the container.
```

Comment:
:question: This is the test for the removal of `maximum_size` from the volumes dict of the listed containers, so maybe make that clear in the test docstring.